lstchain/tools/lstchain_data_create_calibration_file.py

```diff
@@ -192,7 +203,13 @@ def start(self):
                     continue
 
                 # if pedestal event
-                if event.r1.tel[self.tel_id].trigger_type == 32:
+                if event.r1.tel[self.tel_id].trigger_type == 32 or (
+                    self.simulation and
+                    np.median(np.sum(event.r1.tel[self.tel_id].waveform[0], axis=1))
+                    < self.minimum_hg_charge_median):
```

Comment:
what is this condition for?

Reply:
This is for the MC events for which I do not have a trigger